content/academy/apify_platform/getting_started/apify_client.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Apify client
+description: Interact with the Apify API in your code without an HTTP client using the apify-client package, which is available for both JavaScript and Python.
+menuWeight: 5
+paths:
+    - apify-platform/getting-started/apify-client
+---
+
+# [](#apify-client) Apify client
+
+Now that you've gotten your toes wet with interacting with the Apify API through raw HTTP requests, you're ready to become familiar with the **Apify client**, which is a package available for both JavaScript and Python that allows you to interact with the API in your code without  explicitly needing to make any GET or POST requests.
+
+This lesson will provide code examples for both Node.js and Python, so regardless of the language you are using, you can follow along!
+
+## [](#installing-and-importing) Installing and importing
+
+If you are going to use the client in Node.js, use this command within one of your projects to install the package through NPM:
+
+```shell
+npm install apify-client
+```
+
+In Python, you can install it from PyPI with this command:
+
+```shell
+pip install apify-client
+```
+
+After installing the package, let's make a file named **client** and import the Apify client like so:
+
+```marked-tabs
+<marked-tab header="Node.js" lang="javascript">
+// client.js
+import { ApifyClient } from 'apify-client';
+</marked-tab>
+<marked-tab header="Python" lang="python">
+# client.py
+from apify_client import ApifyClient
+</marked-tab>
+```
+
+## [](#running-an-actor) Running an actor
+
+In the last lesson, we ran the **adding-actor** and retrieved its dataset items. That's exactly what we're going to do now; however, by using the Apify client instead.
+
+Before we can use the client though, we must create a new instance of the `ApifyClient` class and pass it our API token from the [**Integrations** page](https://console.apify.com/account?tab=integrations) on the Apify Console:
+
+```marked-tabs
+<marked-tab header="Node.js" lang="javascript">
+const client = new ApifyClient({
+    token: 'YOUR_TOKEN',
+});
+</marked-tab>
+<marked-tab header="Python" lang="python">
+client = ApifyClient(token='YOUR_TOKEN')
+</marked-tab>
+```
+
+> If you are planning on publishing your code to a public Github/Gitlab repository or anywhere else online, be sure to set your API token as en environment variable, and never hardcode it directly into your script.
+
+Now that we've got our instance, we can point to an actor using the `client.actor()` function, then call the actor with some input with the `.call()` function.
```

Comment:
Might be worth mentioning that the input is the first parameter and also link to the function signature/docs.